libs/rtemodel/src/RteProject.cpp

```diff
@@ -436,14 +436,18 @@ void RteProject::AddCprjComponents(const Collection<RteItem*>& selItems, RteTarg
   // update file versions
   for (auto [instanceName, fi] : m_files) {
     string version;
+    auto rteFile = fi->GetFile(target->GetName());
     auto itver = configFileVersions.find(instanceName); // file in cprj can be specified by its instance name
     if (itver == configFileVersions.end())
       itver = configFileVersions.find(fi->GetName()); // or by original name
     if (itver != configFileVersions.end()) {
       version = itver->second;
+    } else {
+      // Fall back to the version noted in the pack
+      version = rteFile->GetVersionString();
```

Comment:
@edriouk You can check out this commit and give the unit-tests a try with out this line and you will see a failure in the test case `RunProjMgr_UpdateRte`.
The wrong base version would be selected if the RTE directory tree does not yet exist.